### PR TITLE
chore: add bundlesize integration

### LINKED ISSUE #835: Add bundlesize
Related to #773, we are going to add bundlesize to now the size in every build. Also, we integrate it in our CircleCI workflow

### PR DESCRIPTION
**Pull request by juanpicado**
**Type:** build

The following has been addressed in the PR:

**Description:**

It adds the test part, is missing the circleCI integration @sergiohgz (feel free to commit on this branch)

Resolves #835

I set some reasonable limits, but we can discuss them.  It's not part of the commitlint, this should be checked only on CI server.

```
$ bundlesize
 PASS  ./static/vendors.591021440b600d88736e.js: 182.17KB < maxSize 200KB (gzip) 

 PASS  ./static/1.591021440b600d88736e.js: 5.09KB < maxSize 20KB (gzip) 

 PASS  ./static/2.591021440b600d88736e.js: 9.47KB < maxSize 20KB (gzip) 

 PASS  ./static/3.591021440b600d88736e.js: 5.09KB < maxSize 20KB (gzip) 

 PASS  ./static/4.591021440b600d88736e.js: 3.83KB < maxSize 20KB (gzip) 

 PASS  ./static/5.591021440b600d88736e.js: 5.01KB < maxSize 20KB (gzip) 

 PASS  ./static/3.style.c3fe21f88c1bc9da2c9a.css: 2.08KB < maxSize 2.1KB (gzip) 

 PASS  ./static/4.style.a6a741f391ea92b3b1e9.css: 932B < maxSize 2.1KB (gzip) 

 PASS  ./static/5.style.e4964c81ec7b931e2d7c.css: 1014B < maxSize 2.1KB (gzip) 

 PASS  ./static/7.style.d8a150ca39aba79b51e5.css: 1.13KB < maxSize 2.1KB (gzip) 

 PASS  ./static/0.style.b124b88bd361db6befbc.css: 40.35KB < maxSize 45KB (gzip) 

 PASS  ./build/api/debug/index.js: 388B < maxSize 5.5KB (gzip) 

 PASS  ./build/api/endpoint/api/dist-tags.js: 743B < maxSize 5.5KB (gzip) 

 PASS  ./build/api/endpoint/api/package.js: 799B < maxSize 5.5KB (gzip) 

 PASS  ./build/api/endpoint/api/ping.js: 161B < maxSize 5.5KB (gzip) 

 PASS  ./build/api/endpoint/api/publish.js: 2.31KB < maxSize 5.5KB (gzip) 

 PASS  ./build/api/endpoint/api/search.js: 1.15KB < maxSize 5.5KB (gzip) 

 PASS  ./build/api/endpoint/api/user.js: 940B < maxSize 5.5KB (gzip) 

 PASS  ./build/api/endpoint/api/whoami.js: 231B < maxSize 5.5KB (gzip) 

 PASS  ./build/api/endpoint/index.js: 878B < maxSize 5.5KB (gzip) 

 PASS  ./build/api/index.js: 1.55KB < maxSize 5.5KB (gzip) 

 PASS  ./build/api/middleware.js: 2.78KB < maxSize 5.5KB (gzip) 

 PASS  ./build/api/web/api.js: 804B < maxSize 5.5KB (gzip) 

 PASS  ./build/api/web/endpoint/api-utils.js: 222B < maxSize 5.5KB (gzip) 

 PASS  ./build/api/web/endpoint/package.js: 1.2KB < maxSize 5.5KB (gzip) 

 PASS  ./build/api/web/endpoint/search.js: 569B < maxSize 5.5KB (gzip) 

 PASS  ./build/api/web/endpoint/user.js: 423B < maxSize 5.5KB (gzip) 

 PASS  ./build/api/web/index.js: 945B < maxSize 5.5KB (gzip) 

 PASS  ./build/config/env.js: 163B < maxSize 5.5KB (gzip) 

 PASS  ./build/index.js: 151B < maxSize 5.5KB (gzip) 

 PASS  ./build/lib/auth-utils.js: 515B < maxSize 5.5KB (gzip) 

 PASS  ./build/lib/auth.js: 2.62KB < maxSize 5.5KB (gzip) 

 PASS  ./build/lib/bootstrap.js: 2.01KB < maxSize 5.5KB (gzip) 

 PASS  ./build/lib/cli.js: 1.02KB < maxSize 5.5KB (gzip) 

 PASS  ./build/lib/config-path.js: 1.25KB < maxSize 5.5KB (gzip) 

 PASS  ./build/lib/config-utils.js: 1.5KB < maxSize 5.5KB (gzip) 

 PASS  ./build/lib/config.js: 1.17KB < maxSize 5.5KB (gzip) 

 PASS  ./build/lib/constants.js: 1.44KB < maxSize 5.5KB (gzip) 

 PASS  ./build/lib/crypto-utils.js: 884B < maxSize 5.5KB (gzip) 

 PASS  ./build/lib/local-storage.js: 5.39KB < maxSize 5.5KB (gzip) 

 PASS  ./build/lib/logger.js: 1.8KB < maxSize 5.5KB (gzip) 

 PASS  ./build/lib/metadata-utils.js: 596B < maxSize 5.5KB (gzip) 

 PASS  ./build/lib/notify.js: 1.31KB < maxSize 5.5KB (gzip) 

 PASS  ./build/lib/plugin-loader.js: 1.18KB < maxSize 5.5KB (gzip) 

 PASS  ./build/lib/search.js: 844B < maxSize 5.5KB (gzip) 

 PASS  ./build/lib/status-cats.js: 1.55KB < maxSize 5.5KB (gzip) 

 PASS  ./build/lib/storage-utils.js: 2.19KB < maxSize 5.5KB (gzip) 

 PASS  ./build/lib/storage.js: 4.49KB < maxSize 5.5KB (gzip) 

 PASS  ./build/lib/up-storage.js: 5.42KB < maxSize 5.5KB (gzip) 

 PASS  ./build/lib/uplink-util.js: 545B < maxSize 5.5KB (gzip) 

 PASS  ./build/lib/utils.js: 4.6KB < maxSize 5.5KB (gzip) 

 PASS  ./build/utils/string.js: 211B < maxSize 5.5KB (gzip) 

 PASS  ./build/utils/user.js: 372B < maxSize 5.5KB (gzip) 
```

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -185,6 +187,28 @@
...
+      "path": "./static/vendor*.js",
+      "maxSize": "200 kB"
+    },
+    {
+      "path": "./static/[0-9].*.js",
+      "maxSize": "20 kB"
+    },
+    {
+      "path": "./static/[1-9].*.css",
+      "maxSize": "2.1 kB"
```
Comment (sergiohgz) [inline package.json]: There is a file with 2.08kB, near the limit. We should grow up to 2.5kB

Comment (juanpicado) [reply package.json]: I was trying to be picky 😆

**Comment by juanpicado** [comment]
Mmmmm ... twice? Any idea why @sergiohgz ?

**Comment by sergiohgz** [comment]
The first is buildsize, the tool we commented on Discord. The second is the real bundlesize we have installed in this PR. 

I think buildsize would not work as it uses CircleCI build artifacts, but we publish as artifacts only the test results.

**Pull request merged**